viewer/packages/tools/src/AxisView/AxisViewTool.test.ts

```diff
@@ -13,15 +13,16 @@ import { defaultAxisBoxConfig } from './types';
 
 describe('AxisViewTool', () => {
   let canvasContainer: HTMLElement;
+  let domSize: { height: number; width: number };
   let viewer: Cognite3DViewer;
 
-  const sdk = new CogniteClient({ appId: 'cognite.reveal.unittest' });
-  mockClientAuthentication(sdk);
-  const context = createGlContext(64, 64, { preserveDrawingBuffer: true });
-  const renderer = new THREE.WebGLRenderer({ context });
-  const domSize = { height: 480, width: 640 };
-
   beforeEach(() => {
+    const sdk = new CogniteClient({ appId: 'cognite.reveal.unittest' });
```

Comment:
From how it was written previously, I assume it could be inside `beforeAll`, not `beforeEach` because context could be the same for each test.

Reply:
I guess it could be in beforeAll. I generally don't use it a lot as beforeEach ensures that no state "bleeds" between test A and B and generally it's fast enough to just re-init for each test.